os/include/tinyara/kmalloc.h

```diff
@@ -121,10 +121,15 @@ extern "C" {
  * (CONFIG_BUILD_KERNEL), the following are declared in stdlib.h and are
  * directly callable.
  */
-
+#ifdef CONFIG_SPIRAM_SUPPORT
```

Comment:
I don't know this file change is related with this commit.

Reply:
Add new config for this general file